packages/kbn-securitysolution-list-utils/src/get_filters/index.test.ts

```diff
@@ -11,106 +11,218 @@ import { getFilters } from '.';
 describe('getFilters', () => {
   describe('single', () => {
     test('it properly formats when no filters passed and "showTrustedApps" is false', () => {
-      const filter = getFilters({}, ['single'], false);
+      const filter = getFilters({}, ['single'], false, false);
 
-      expect(filter).toEqual('(not exception-list.attributes.list_id: endpoint_trusted_apps*)');
+      expect(filter).toEqual(
+        '(not exception-list.attributes.list_id: endpoint_trusted_apps*) AND (not exception-list.attributes.list_id: endpoint_event_filters*)'
+      );
     });
 
     test('it properly formats when no filters passed and "showTrustedApps" is true', () => {
-      const filter = getFilters({}, ['single'], true);
+      const filter = getFilters({}, ['single'], true, false);
```

Comment:
This API is getting difficult to read, once we have something such as `true, false` on the end mixed with an object and an array. Maintainers and others will have a hard time knowing which true and false belongs to what without reading the source function.

Could we maybe just use an object so that we have named input variables?

Then this API would look something like this and be easier to read in the code base that the second false indicates that it is a `showEventFilters` vs `showTrustedApps`:

Usually when people get to 2-3 parameters it is easier to read named parameters than guessing what each `true, false, etc...` means.

```ts
 getFilters({
  filters: {},
  namespaceTypes: ['single'],
  showTrustedApps: true,
  showEventFilters: false,
})
```

Is it possible to change the API at this point to use an